samples/same54xprov2/basic-sample/include/app_config.h

```diff
@@ -0,0 +1,25 @@
+//
+// Copyright: Avnet 2021
+// Created by Nik Markovic <nikola.markovic@avnet.com> on 4/19/21.
+//
+
+#define IOTCONNECT_APP_USE_WEATHER_CLICK
+
+#ifndef APP_CONFIG_H
+#define APP_CONFIG_H
+
+#define IOTCONNECT_CPID "avtds"
+#define IOTCONNECT_ENV  "avnetpoc"
+
+// If using the ENABLE_DDIM_PKCS11_ATCA_DRIVER_SAMPLE define, the DUID will be automatically generated
+//#define IOTCONNECT_DUID "same54xpro" // you can supply a custom device UID, or...
```

Comment:
Would using a specific device id be easier for a demo? IMHO it doesn't make much difference if it is  IOTCONNECT_DUID or DUID_PREFIX as long as it is documented,  but DUID_PREFIX has the advantage of multiple devices not needing a new binary compiled  :)